.travis.yml

```diff
@@ -55,13 +52,21 @@ before_script:
       sudo apt-add-repository ppa:ondrej/mysql-5.6 -y;
       sudo apt-get update;
       sudo apt-get install mysql-server-5.6 mysql-client-5.6;
-      mysql -uroot -e 'SET @@global.sql_mode = NO_ENGINE_SUBSTITUTION; CREATE DATABASE magento_integration_tests;';
+      sudo mysql -e 'SET @@global.sql_mode = NO_ENGINE_SUBSTITUTION; CREATE DATABASE magento_integration_tests;';
       mv dev/tests/integration/etc/install-config-mysql.travis.php.dist dev/tests/integration/etc/install-config-mysql.php;
       fi"
+
+  # try to avoid mysql has gone away errors
```

Comment:
This try was not successful if I remember build results correctly.

Currently integration tests on mainline are green, so, please revert these changes.